README.md

```diff
@@ -174,6 +174,7 @@ configuration options.
     * processes
     * kernel (/proc/stat)
     * kernel (/proc/vmstat)
+    * linux_sysctl_fs (/proc/sys/fs)
```

Comment:
Yeah I don't care for kernel as a name, but we should think about how to make the transition smooth to 2.0, so everything doesn't just get renamed when you switch versions.  Maybe we should add the new names as aliases, so people can switch now and then for the real 2.0 all we do is drop support.  Obviously thats a lot of work for the size of this change... something to think about though.